### PR TITLE
Added performance comparison table in gatling build page

### PR DESCRIPTION
**Pull request by yanickbelanger**
screen shot 2015-08-14 at 10 56 52 am

### CODE REVIEW

**Comment by yanickbelanger** [comment]
@gjacquet Please review and merge.

File: src/main/java/io/gatling/jenkins/DetailedRequestReport.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+public class DetailedRequestReport {
+
+  private Map<String, DetailedRequestReportContent> contents;
+
+  public void setContents(Map<String, DetailedRequestReportContent> contents) {
+    this.contents = contents;
+  }
+  
+  public HashMap<String, RequestReport> getReports() {
```
Comment (gjacquet) [inline src/main/java/io/gatling/jenkins/DetailedRequestReport.java]: `public Map<String, RequestReport> getReports` ?

File: src/main/java/io/gatling/jenkins/PerformanceDelta.java [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+import java.util.Map;
+
+import hudson.model.AbstractBuild;
+
+/**
+ * Created by yanick.belanger on 8/13/15.
+ */
+public class PerformanceDelta {
+
+	private List<RequestPerformanceDelta> deltas;
```
Comment (gjacquet) [inline src/main/java/io/gatling/jenkins/PerformanceDelta.java]: If we ever want to contribute that back we need to follow their conventions and indent settings.

File: src/main/java/io/gatling/jenkins/PerformanceDelta.java [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+			return Float.compare(other.getRelativeDeltaForComparison(), this.getRelativeDeltaForComparison());
+		}
+
+		private String formatValue(Long value) {
+			return value == null ? "N/A" : (value + "ms");
+		}
+
+		private float getRelativeDeltaForComparison() {
+			Float relativeDelta = getRelativeDelta();
+			return relativeDelta == null ? Float.MIN_VALUE : relativeDelta;
```
Comment (gjacquet) [inline src/main/java/io/gatling/jenkins/PerformanceDelta.java]: Why min value?

File: src/main/java/io/gatling/jenkins/DetailedRequestReport.java (hunk 1 above)
Comment (yanickbelanger) [inline src/main/java/io/gatling/jenkins/DetailedRequestReport.java]: Oops, I blame the "extract method" function in IntelliJ! Will fix it.

File: src/main/java/io/gatling/jenkins/PerformanceDelta.java (hunk 2 above)
Comment (yanickbelanger) [inline src/main/java/io/gatling/jenkins/PerformanceDelta.java]: True, will fix.

File: src/main/java/io/gatling/jenkins/PerformanceDelta.java (hunk 3 above)
Comment (yanickbelanger) [inline src/main/java/io/gatling/jenkins/PerformanceDelta.java]: If no delta info is available, we simply want them to be at the end of the list.

**Comment by gjacquet** [comment]
LGTM

**Pull request merged**